tests/test_history.py

```diff
@@ -557,6 +558,277 @@ def handle_data(context, data):
         self.assertEquals(139.36946942498648, last_prices[oldest_dt])
         self.assertEquals(180.15661995395106, last_prices[newest_dt])
 
+    def test_history_in_bts_price_days(self):
+        """
+        Test calling history() in before_trading_start()
+        with daily price bars.
+        """
+        algo_text = """
+from zipline.api import history
+
+def initialize(context):
+    context.first_day = True
+
+def before_trading_start(context, data):
+    if not context.first_day:
+        prices_bts = history(bar_count=3, frequency='1d', field='price')
+        context.prices_bts = prices_bts
+
+def handle_data(context, data):
+    prices_hd = history(bar_count=3, frequency='1d', field='price')
+    context.prices_hd = prices_hd
+    context.first_day = False
```

Comment:
Sure, but then calling it `first_day` won't make sense, since we'll enter `handle_data` on the first trading day, yet `first_day == False`. Maybe call the flag `first_bts_call` intead.